src/common/components/supportSidebar.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-console */
```

Comment:
you need to adapt to the eslint rules, remove these comment lines and fix the eslint warns